src/domain/datasets/domain/Cargo.toml

```diff
@@ -25,7 +25,9 @@ doctest = false
 database-common = { workspace = true }
 internal-error = { workspace = true }
 opendatafabric = { workspace = true, default-features = true, features = [
-    "sqlx",
+    "sqlx-mysql",
```

Comment:
Why do we depend on sqlx from domain-level crate? I don't even see tests in this crate so its seems that it's not needed even as a `dev-dependency`
@rmn-boiko @zaychenko-sergei